radar/dev/angular.yaml

```diff
@@ -3,13 +3,20 @@ logo: https://angular.io/assets/images/logos/angular/angular.png
 blip:
   - date: 2019-03-29
     ring: ADOPT
+  - date: 2020-10-18
+    ring: HOLD
 description: |
   Angular is a platform for building mobile and desktop web applications.
 rationale: |
   Angular is a complete web application framework running on TypeScript. With
   a convention over configuration approach and TypeScript as a requirement it allows
   for an easier transition from backend to frontend and with a complete set
   of tooling it enables us to build large scale web applications.
+
+  We currently have both React and Angular in ADOPT, but we do not intend to support both
```

Comment:
Can we change the tense here to something like "Formerly, we had both React and Angluar in Adopt ..."? That would feel more natural when this blip has been moved to hold.

Reply:
done